1_The_Physical/Level_2_Stateful_Guess_Point/Shared/TicTacToe/src/tic-tac-toe.spec.ts

```diff
@@ -1,17 +1,25 @@
-import { TicTacToe } from "./tic-tac-toe"
+import { TicTacToe } from "./tic-tac-toe";
 
-describe('Given a game is started', () => {
+describe("Given a game is started", () => {
   it("Then a game instance should create", () => {
-    const game = TicTacToe.start()
+    const game = TicTacToe.start();
 
-    expect(game).toBeInstanceOf(TicTacToe)
-  })
-})
+    expect(game).toBeInstanceOf(TicTacToe);
+  });
+});
 
-describe('Given a game is started', () => {
+describe("Given a game is started", () => {
   it("Then is 9 spaces on the board", () => {
-    const game = TicTacToe.start()
+    const game = TicTacToe.start();
 
-    expect(game.getAmountOfBoardSquares).toEqual(9)
-  })
-})
+    expect(game.getAmountOfBoardSquares).toEqual(9);
```

Comment:
So far so good! Love the baby steps here. I have a module planned for this, but you want to avoid using the 'get' keyword and prefer `getAmountOfBoardSquares()`. It can confuse whether you're referring to a property or a method, using the `get` keyword. I've seen some pretty nasty bugs caused by that. Make sense?